spec/models/setting_spec.rb

```diff
@@ -54,6 +54,7 @@
       expect(described_class.alert_via_sentry?).to be true
```

Comment:
@skdejong I think you could add a test for opponent_organisations? to the "when there is no setting record" and "when a record already exists with non-default values" contexts in this spec - give me a shout if you want to pair 🙂

Reply:
Thank you, Katharine. I have added these tests now.